### PR TITLE
Add ci supportfix

### PR DESCRIPTION
**Pull request by FrankD412**
Addresses the issue of support Python versions and updates tox settings to appropriately mirror the selection of Python 2.7 and 3.6.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+sudo: false
+language: python
+python:
+    - "2.6"
```
Comment (IanLee1521) [inline .travis.yml]: This list should match what is in the tox.ini file (you probably can do this entire list, but you may need to fine tune it.

Comment (IanLee1521) [inline .travis.yml]: And setup.py

Comment (FrankD412) [inline .travis.yml]: I fixed .travis.yaml -- but for setup.py are you referring to using the ```python_requires``` entry?

File: setup.py [hunk 2]
```diff
@@ -23,9 +23,9 @@
       classifiers=[
         'Development Status :: 3 - Alpha',
         'Intended Audience :: Developers',
-        'Operating System :: OS Independent',
+        'Operating System :: Unix',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 2.7',
```
Comment (IanLee1521) [inline setup.py]: These classifiers should be updated to include all relevant python versions.

**Pull request merged**